react-components/stories/Architecture.stories.tsx

```diff
@@ -48,16 +52,23 @@ export const Main: Story = {
   }
 };
 
-function StoryContent({ addModelOptions }: { addModelOptions: AddModelOptions }): ReactElement {
+function StoryContent({
+  cadModelOptions,
+  pointCloudModelOptions
+}: {
+  cadModelOptions: AddModelOptions;
+  pointCloudModelOptions: AddModelOptions;
+}): ReactElement {
   const renderTarget = useRenderTarget();
   return (
     <>
       <CadModelContainer
-        addModelOptions={addModelOptions}
+        addModelOptions={cadModelOptions}
         onLoad={(_model: CogniteCadModel) => {
           renderTarget.onStartup();
         }}
       />
+      <PointCloudContainer addModelOptions={pointCloudModelOptions} />
     </>
```

Comment:
Is it easy to change this to `Reveal3DResources`? I think I'd prefer that, but not necessary if it's hard

Reply:
To use `Reveal3DResources` for loading local models would require some effort to enable the support, I would refer to do it for  later improvements